### PR TITLE
Auto-update script and legal improvements

### LINKED ISSUE #1: Automate build/deploy
pdfium-binaries are re-built every Monday, automatically (with Travis and Appveyor). Would be nice to make pypdfium follow the same pattern and automatically build and release (every Tuesday?).

Script should do the following:
1. Find out the latest tag
   ```bash
   git ls-remote --heads https://github.com/bblanchon.pdfium-binaries.git | grep -ohP 'chromium/\d+'  | tail -n1`
   ```
2. build download URLs for each platform, download binaries
3. extract binaries
4. run `ctypesgen` on extracted lib and headers
5. commit binaries and python wrapper
6. build wheel
7. deploy to PyPi

### LINKED ISSUE #2: Add pdfium LICENSE
Please include the pdfium LICENSE file which also comes in the archives that you include the libraries from. This is mandated both by the BSD-3-Clause and by the Apache 2.0 license that pdfium is licensed under.

### PR DESCRIPTION
**Pull request by mara004**
This PR adds a script named `update_bindings.py` to automate the process of retrieving the latest PDFium binaries, running `ctypesgen` and building a wheel, which resolves #1.
Moreover, it also fixes #2 by adding the PDFium license file, following the [reuse](https://reuse.software/tutorial/) standard.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,9 +21,9 @@ def readme():
       long_description=readme(),
       long_description_content_type="text/markdown",
       keywords='Python PDFium',
-      url='https://github.com/YinlinHu/pypdfium',
-      author='Yinlin Hu',
-      author_email='huyinlin@gmail.com',
+      url='https://github.com/mara004/pypdfium',
+      maintainer='Manuel Geißer',
+      maintainer_email='geisserml@gmail.com',
```
Comment (bgermann) [inline setup.py:26]: Are you sure you want to include url and maintainer changes in the pull request?

**Comment by bgermann** [comment]
Imho, this pull request should be rebased to have a consistent git author, no merge commits, and more meaningful git commit messages (e.g., not a lot of repetitive "Update README.md").

**Comment by mara004** [comment]
I'll likely close this PR - I didn't really believe it to be merged when I opened it. PyPDFium has been unmaintained for quite a while now. I plan to work on my own branch and publish a new PyPI package. The many "Update README.md" messages are because I already started with this transformation - It doesn't make sense to write a description for every tiny change anyway.

File: setup.py (hunk 1 above)
Comment (mara004) [reply setup.py:26]: I did this in my repository, not thinking of the pull request

**Comment by mara004** [comment]
> have a consistent git author

May I ask what you mean by that? Probably the missing signatures for some commits?

**Comment by bgermann** [comment]
Some commits have author "mara004 <65915611+mara004@users.noreply.github.com>", some have "Manuel Geißer <redacted>".

**Comment by mara004** [comment]
Ah, I see, that's because I did some commits locally with the git cli.

**Comment by YinlinHu** [comment]
@mara004 Great job. Just merged it.

**Comment by bgermann** [comment]
Have you noticed that the README title and the maintainer name/email changed? I guess these are things that you do not want from @mara004's version.

**Comment by mara004** [comment]
@YinlinHu thanks, but @bgermann is right that it should not have been merged in this state. I believed PyPDFium was really unmaintained and didn't think of the pull request to get actually merged, so I already started to prepare for a fork with the pr still open, which was silly. Sorry about that. Should I open a new pull request to correct this blunder?

**Comment by mara004** [comment]
I corrected it now, as I received commit rights to this repository.

**Comment by mara004** [comment]
I will nevertheless keep working on my own branch since I do not have access to the PyPI package

**Comment by YinlinHu** [comment]
@mara004, please give me your user name on pypi.org. I can set you as the collaborator.

**Pull request closed without merging**